_posts/2024-02-14-provisioning-ubuntu-autoinstall.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Provisioning Ubuntu Autoinstall
+date: 2024-02-14 09:41:32
+author: Maximilian Kolb
+tags:
+- foreman
+- provisioning
+- ubuntu
+- autoinstall
+---
+
+# Provisioning Ubuntu Autoinstall
```

Comment:
I'd drop this, you already have the title at the top of the page.

Reply:
Dropped.